src/Symfony/Component/Security/Core/composer.json

```diff
@@ -19,7 +19,8 @@
         "php": ">=8.1",
         "symfony/event-dispatcher-contracts": "^2.5|^3",
         "symfony/service-contracts": "^2.5|^3",
-        "symfony/password-hasher": "^5.4|^6.0"
+        "symfony/password-hasher": "^5.4|^6.0",
+        "symfony/deprecation-contracts": "^2.5|^3"
```

Comment:
alpha order still :) (it should be after "php")

Reply:
Though that `composer require` takes care of that.  :eyes: